digitalocean/baseapi.py

```diff
@@ -46,7 +46,7 @@ class BaseAPI(object):
     end_point = "https://api.digitalocean.com/v2/"
 
     def __init__(self, *args, **kwargs):
-        self.token = ""
+        self.token = os.environ.get("DIGITALOCEAN_ACCESS_TOKEN")
```

Comment:
Why not using `os.getenv` with a default value `""`?

Reply:
No particular reason, only my lack of experience with Python :wink: